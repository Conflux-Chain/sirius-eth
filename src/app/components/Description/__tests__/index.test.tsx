import React from 'react';
import { render } from '@testing-library/react';
import { Description } from '..';

describe('<Description  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(
      <Description title={'title'}>content</Description>,
    );
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
