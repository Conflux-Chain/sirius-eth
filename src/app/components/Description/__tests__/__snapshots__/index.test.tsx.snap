// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Description  /> should match snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  min-height: 3.2857rem;
  line-height: 3.2857rem;
  border-bottom: 1px solid #e8e9ea;
}

.c0.no-border {
  border-bottom: none;
}

.c0 .left {
  padding: 0.8571rem 0;
  line-height: calc(3.2857rem - 1.7143rem);
  width: 25%;
  min-width: 160px;
  max-width: 260px;
  color: #74798c;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
}

.c0 .right {
  padding: 0.8571rem 0;
  line-height: calc(3.2857rem - 1.7143rem);
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  color: #282d30;
}

.c0.tiny .left,
.c0.tiny .right {
  padding: 0.5rem 0;
}

.c0.verticle {
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c0.verticle .left {
  width: 100%;
}

.c0.verticle .right {
  padding-top: 0;
}

.c0.small .left {
  width: 10rem;
}

@media (max-width:768px) {
  .c0 .right {
    padding-top: 0;
  }
}

@media (max-width:768px) {
  .c0 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
  }
}

<div
  class="c0 description medium"
>
  <div
    class="left"
  >
    title
  </div>
  <div
    class="right"
  >
    content
  </div>
</div>
`;
